core/src/scss/utilities/variables/core/_core-color-maps.scss

```diff
@@ -37,10 +37,10 @@ $core-colors: (
   text--high-contrast:          #000,
   text--reverse:                $color-white,
 
+  link:                         $color-bright-blue,
+  link--hover:                  $color-black,
```

Comment:
I chose to replace `link` and `link--hover` with `link-inline` and `link-inline--hover` to be more explicit about the application. I don't want to have both, so let's decide. I prefer the `-inline` variants, in which case please use those. But if you feel strongly that `link` and `link--hover` are better, then please delete the ones I added and change all references to the `-inline` map entries to match

Reply:
I like having a clean default `link` color without any modifiers since we have clean `bg`, `text` colors. Also, the `-inline` makes me wonder if it only applies to `display:inline` links. For example, the links in the vertical link list will probably take on this default link color, but it might be a bit strange if I apply the color `link-inline` even though the link might be a block element.

Reply:
Ok, then please delete the `-inline` options and update all references to them to refer to the shorter names.

Reply:
Done! Added a link--alt and link--alt-hover for cases like the secondary button link style. I feel like the bright red link will appear on some sites in some other ways so I make that the link--alt color.

Reply:
Ok. I introduced an error in the doc for `@function color()`, so I fixed that.

Reply:
Thanks!!